github/ci/prow-deploy/molecule/default/vars/kubevirtci-testing.yml

```diff
@@ -6,3 +6,7 @@ kubeconfig_path: '/root/.kube/config'
 secrets_dir: '{{ project_infra_root }}/github/ci/prow-deploy/kustom/overlays/kubevirtci-testing/secrets/'
 
 kind_cluster_config: '/workspace/cluster.yaml'
+
+SRIOVNodes:
+- name: node01
```

Comment:
nit (not mandatory),
maybe better to call it node_name?

Reply:
Given that the top-level element is called `SRIOVNodes` calling one of the child fields `node_*` would be redundant as I see it (given that all the children are nodes), wdyt?

Reply:
Np, understood,
just cause when it refereed its item.name
but all is good, thanks